alacritty/src/input.rs

```diff
@@ -366,23 +366,66 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
         }
     }
 
-    pub fn normal_mouse_report(&mut self, button: u8) {
+    pub fn utf8_mouse_report(&mut self, button: u8) {
         let (line, column) = (self.ctx.mouse().line, self.ctx.mouse().column);
 
-        if line < Line(223) && column < Column(223) {
-            let msg = vec![
+        if line < Line(2015) && column < Column(2015) {
+            let mouse_pos_encode = |pos: usize| -> (u8, u8) {
+                let pos = 32 + 1 + pos;
+                let first = 0xC0 + (pos >> 6);
```

Comment:
Should improve readability a bit.

```suggestion
                let first = 0xC0 + pos / 64;
```

Reply:
I don't know about this one, I prefer bit operations when manipulating with bits.

Reply:
We're just dividing by 64 though, we're not `manipulating with bits`.

For the bitwise and it makes sense, but for this it's much, much more difficult to clearly understand the code just by reading it.